Sources/Commands/SwiftTestTool.swift

```diff
@@ -136,8 +146,17 @@ public class SwiftTestTool: SwiftTool<TestToolOptions> {
                 print(test.specifier)
             }
 
+        case .generateLinuxMain:
+          #if os(Linux)
```

Comment:
should this also be ` #if !os(macOS)`?

Reply:
Yeah, that will be more accurate. Thanks!